src/components/by-user-view.jsx

```diff
@@ -11,95 +11,90 @@ import IssueList from './issue-list';
 import Board from './board';
 
-/* Copy-pasta from MilestonesView */
-
-const KanbanColumn = React.createClass({
-  render() {
-    const {login, cards, primaryRepoName, columnRegExp} = this.props;
-
-    const issueComponents = _.map(cards, (card) => {
-      return (
-        <Issue
-          key={card.key()}
-          primaryRepoName={primaryRepoName}
-          card={card}
-          columnRegExp={columnRegExp}
-          />
-      );
-    });
-
-    const heading = (
-      <span className='user-title'>
-        <UserIcon/>
-        {login}
-      </span>
-    );
+function KanbanColumn(props) {
+  const {login, cards, primaryRepoName, columnRegExp} = props;
 
+  const issueComponents = _.map(cards, (card) => {
     return (
-      <div key={login} className='kanban-board-column'>
-        <IssueList title={heading}>
-          {issueComponents}
-        </IssueList>
-      </div>
+      <Issue
+        key={card.key()}
+        primaryRepoName={primaryRepoName}
+        card={card}
+        columnRegExp={columnRegExp}
+        />
     );
-
-  }
-});
-
-
-const UsersView = React.createClass({
-  render() {
-    const {repoInfos, columnData, cards, columnRegExp} = this.props;
-    const [{repoName}] = repoInfos; // primaryRepoName
-
-    let sortedCards = FilterStore.filterAndSort(cards, true);
-
-    let kanbanColumnCount = 0; // Count the number of actual columns displayed
-
-    const kanbanColumns = _.map(columnData, (login) => {
-      // If we are filtering by a kanban column then only show that column
-      // Otherwise show all columns
-      const columnCards = _.filter(sortedCards, (card) => {
-        return card.issue.owner && card.issue.owner.login === login || card.issue.user.login === login;
-      });
+  });
+
+  const heading = (
+    <span className='user-title'>
+      <UserIcon/>
+      {login}
+    </span>
+  );
+
+  return (
+    <div key={login} className='kanban-board-column'>
+      <IssueList title={heading}>
+        {issueComponents}
+      </IssueList>
+    </div>
+  );
+
+}
+
+function UsersView(props) {
+  const {repoInfos, columnData, cards, columnRegExp} = props;
+  const [{repoName}] = repoInfos; // primaryRepoName
+
+  let sortedCards = FilterStore.filterAndSort(cards, true);
+
+  let kanbanColumnCount = 0; // Count the number of actual columns displayed
+
+  const kanbanColumns = _.map(columnData, (login) => {
+    // If we are filtering by a kanban column then only show that column
+    // Otherwise show all columns
+    const columnCards = _.filter(sortedCards, (card) => {
+      return card.issue.owner && card.issue.owner.login === login || card.issue.user.login === login;
+    });
 
 
-      // Show the column when:
-      // isFilteringByColumn = label (the current column we are filtering on)
-      // !isFilteringByColumn && (!getShowEmptyColumns || columnCards.length)
-
-      kanbanColumnCount++; // Count the number of actual columns displayed
-      /*HACK: Column should handle milestones */
```

Comment:
this set of comments is duplicated from `by-milestone-view.jsx`.

should this handle milestones in `by-user-view.jsx`?

Reply:
Looks like when the author reused the code from `by-milestone-view.jsx`, he forgot to delete the unrelated comment.